protos/feast/core/FeatureReference.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package feast.core;
+
+option go_package = "github.com/feast-dev/feast/sdk/go/protos/feast/core";
+option java_outer_classname = "FeatureReferenceProto";
+option java_package = "feast.proto.core";
+
+import "feast/core/Feature.proto";
+
+
+// A reference to features in a feature view
+message FeatureReference {
+  // The feature view name
+  string feature_view_name = 1;
+
+  // The features of the feature view that are a part of the feature reference.
+  repeated FeatureSpecV2 feature_columns = 2;
```

Comment:
I am a bit confused. A feature ref is a reference to a single feature. Does this imply a reference to a feature view? like a feature view ref?

Also, should we dedupe it with https://github.com/feast-dev/feast/blob/bcf925500d3aaf8711aef7e2f00cf8fafd31b2ee/protos/feast/serving/ServingService.proto#L52?

Reply:
TIL about this message, but this is somewhat different from what we want in the FeatureViewReference. I think the structure of the new message is required, unless you disagree.